### PR TITLE
Add a PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by alieander**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Reason for Change
+===================
+Describe the big picture of your changes here to communicate why this pull
```
Comment (adarsh) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Maybe bullets here also?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* As a concise list of items
+
+Risks
+=====
+Describe any risks you see to existing systems this fix or feature might have
+if those systems where to incorporate this change.
+
+Checklist
+=========
+
```
Comment (adarsh) [inline .github/PULL_REQUEST_TEMPLATE.md]: Style: extra newline?

**Comment by adarsh** [comment]
Nice! Consider also adding a `.gitmessage` file which either symlinks to this one or is separate and a subset of the headings.

**Comment by alieander** [comment]
> Nice! Consider also adding a .gitmessage file which either symlinks to this one or is separate and a subset of the headings. 

This is a good idea! I will do it in another PR so that I can test this PR!

**Pull request merged**